ragna/deploy/_ui/modal_configuration.py

```diff
@@ -224,19 +221,19 @@ def advanced_config_ui(self):
                         self.config.param.chunk_size,
                         name="Chunk Size",
                         bar_color=ui.MAIN_COLOR,
-                        stylesheets=[ui.SS_LABEL_STYLE],
+                        css_classes=["modal_configuration_int_slider"]
+                        + (["disabled"] if disabled_source_storage else []),
```

Comment:
Nit: I really dislike adding lists. Feel free to ignore.

```suggestion
                        css_classes=["modal_configuration_int_slider" ,*["disabled"] if disabled_source_storage else []],
```